.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- See https://github.com/ucsb-cs156-m23/m23-10am-4-NOTES/blob/main/masterDoc.md for more info-->
+## Overview
+<!--A paragraph of the PR and related content-->
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## Screenshots (Optional)
+<!--Necessary screenshots and any necessary captions here. Delete if not needed.-->
+![Generic placeholder image](https://picsum.photos/640/480)
+
+## Feedback Request (Optional)
+<!--Anywhere specific you want reviewers to take a look at and give suggestions. Delete if not needed.-->
+None needed. I am the best.
```

Comment:
I wonder if we could rephrase this.  

Keep in mind that if your team's repo is chosen, this becomes the new production version of HappyCows.  So, it's important to keep this professional.

Reply:
No worries. It was meant as a light-hearted placeholder text for the team to change or remove. But I'll remove the second sentence and change the other placeholders as well.